tests/fsharpqa/testenv/src/ILComparer/ILComparer.fsproj

```diff
@@ -2,7 +2,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net45</TargetFramework>
+    <TargetFramework>net472</TargetFramework>
```

Comment:
does this need to be netframework, or could it also be netcoreapp?

Reply:
> does this need to be netframework, or could it also be netcoreapp?

This is fine as is.  Vlads library does il compare better than fsharpqa.  Eventually this code will die.